pkg/diagnostics/networkpod/pod.go

```diff
@@ -148,6 +148,13 @@ func (d CheckPodNetwork) checkConnection(pods1, pods2 []kapi.Pod, warnMsg string
 			}
 		}
 	}
+
+	if !sameNamespace && !strings.Contains(warnMsg, "global to non-global projects") {
```

Comment:
Why don't you warn in that case? Is there some better way of testing for that than doing a substring check on the warning message?